CMakeLists.txt

```diff
@@ -92,7 +91,37 @@ mark_as_advanced(
 
 
 if(AF_WITH_GRAPHICS AND NOT AF_USE_SYSTEM_FORGE)
-  include(build_forge)
+    set(ArrayFireInstallPrefix ${CMAKE_INSTALL_PREFIX})
+    set(ArrayFireBuildType ${CMAKE_BUILD_TYPE})
+    set(CMAKE_INSTALL_PREFIX ${ArrayFire_BINARY_DIR}/extern/forge/package)
+    set(CMAKE_BUILD_TYPE Release)
+    set(FG_BUILD_EXAMPLES OFF)
+    set(FG_BUILD_DOCS OFF)
+    set(FG_WITH_FREEIMAGE OFF)
+    add_subdirectory(extern/forge)
+    mark_as_advanced(
+        FG_BUILD_EXAMPLES
+        FG_BUILD_DOCS
+        FG_WITH_FREEIMAGE
+        FG_USE_WINDOW_TOOLKIT
+        FG_USE_SYSTEM_CL2HPP
+        FG_ENABLE_HUNTER
+        glfw3_DIR
+        glm_DIR
+        glbinding_DIR
+        )
+    add_custom_command(
```

Comment:
`install` target doesn't run automatically in submodule targets from what I have experienced. For the reason mentioned in previous comment, I needed a way to run `install` script so that dependencies from forge in once place.